src/setup.js

```diff
@@ -198,7 +190,7 @@ if (currentScript) {
         env:                currentScript.getAttribute('data-env'),
         stage:              currentScript.getAttribute('data-stage'),
         apiStage:           currentScript.getAttribute('data-api-stage'),
-        stageUrl:           currentScript.getAttribute('data-stage-url'),
+        stageUrl:           isPayPalDomain() ? currentScript.getAttribute('data-stage-url') : undefined,
```

Comment:
Good to see a single place for setting `stageUrl` with `data-stage-url` and enforcing the regex domain check.